docs/getting-started/which-permissions-does-codacy-need-from-my-account.md

```diff
@@ -206,8 +201,13 @@ If you need to use an integration that you have previously revoked, log in again
 
 ## Why does Codacy ask for permission to create SSH keys?
 
-When you add a private repository to Codacy, Codacy uses the integration with your Git provider to create a new SSH key on the repository. Codacy then uses that SSH key every time it needs to clone the repository.
+!!! info "This section applies only to GitLab and Bitbucket"
 
-**Codacy only adds read-only SSH keys** and can't access any of your existing SSH keys. You have full control over which organizations and repositories Codacy is authorized to access, and you can also [revoke the keys created by Codacy at any time](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/reviewing-your-deploy-keys). Codacy doesn't change the contents or member privileges of any repository you authorize it to analyze.<!--NOTE Even though this section applies to all Git providers, we're only providing a link to the GitHub docs for the sake of simplicity.-->
+On GitLab and Bitbucket organizations, when you add a private repository to Codacy, Codacy uses the integration with your Git provider to create a new SSH key on the repository. Codacy then uses that SSH key every time it needs to clone the repository.
+
+**Codacy only adds read-only SSH keys** and can't access any of your existing SSH keys. You have full control over which organizations and repositories Codacy is authorized to access. Codacy doesn't change the contents or member privileges of any repository you authorize it to analyze.
 
 We understand the desire for security and privacy and find that the SSH protocol is preferable to HTTPS as it separates Codacy's access rights from the one of the users.
+
+!!! tip
```

Comment:
@andrzej-janczak Can you please validate if these are the most indicated links to GitLab and Bitbucket documentation?

Reply:
hm for GL we use deploy keys that are located here
doc: https://docs.gitlab.com/ee/user/project/deploy_keys/index.html#view-deploy-keys

![Screenshot 2023-11-14 at 09 52 54](https://github.com/codacy/docs/assets/122018786/6e731aee-5c5c-4993-ae86-7f17e20da2ca)

for BB looks like the best place would be: 
https://support.atlassian.com/bitbucket-cloud/docs/configure-repository-settings/ 
![Screenshot 2023-11-14 at 09 55 23](https://github.com/codacy/docs/assets/122018786/92a4e05e-a294-49d1-8f7d-fc9f513986e9)

this is also like a deploy key (read only ssh key per repo) but not using "deploy" word.

Reply:
Updated!